tests/integration/targets/ios_logging_global/tests/cli/deleted.yaml

```diff
@@ -0,0 +1,22 @@
+---
+- debug:
+    msg: Start Deleted integration state for ios_logging ansible_connection={{
+      ansible_connection }}
+
+- include_tasks: _remove_config.yaml
+- include_tasks: _populate_config.yaml
+
+- block:
```

Comment:
can u include delete operation task for few params as well in addition to delete_all